draft-ietf-tokbind-https-09.xml

```diff
@@ -391,16 +389,19 @@
         the encompassing application. 
         </t>
         
-        <t>The scoping for those Token Binding key pairs generated by Web browsers in 
-        the context of the first-party and federation use cases defined in this 
-        specification (below), and to be used for binding HTTP cookies MUST be at the 
-        granularity of "effective top-level domain (public suffix) + 1" (eTLD+1), 
-          i.e., at the same granularity at which cookies can be set 
-          (see <xref target="RFC6265"/>).  Key pairs used to bind other application
+        <t>The scoping of Token Binding key pairs generated by Web browsers for use in 
+        first-party and federation use cases defined in this 
+        specification (<xref target="federation"/>, below), and intended for 
+        binding HTTP cookies, MUST be at the 
```

Comment:
At IETF 98, we agreed to say that browsers absolutely cannot exceed the scope of eTLD+1, but can use a narrower scope if cookies are scoped more narrowly.